include/functions.h

```diff
@@ -1517,10 +1486,10 @@ s16 ActorCutscene_GetCurrentIndex(void);
 ActorCutscene* ActorCutscene_GetCutscene(s16 index);
 s16 ActorCutscene_GetAdditionalCutscene(s16 index);
 s16 ActorCutscene_GetLength(s16 index);
-// void func_800F2138(void);
+s16 func_800F2138(s16);
```

Comment:
```suggestion
s16 func_800F2138(s16 arg0);
```
Unless you know what this argument is